lua/entities/gmod_tardis/modules/sv_power.lua

```diff
@@ -0,0 +1,7 @@
+ENT:AddHook("CanDemat", "power-disable", function(self)
+    return self.interior:GetData("power-state") or false
```

Comment:
don't return true here, only return false or nil or this may have unintended side effects by allowing demat when it shouldnt

Reply:
Only returning on false now